motor.cpp

```diff
@@ -23,7 +23,15 @@ void Motor::stop(int motor, int motor2, int motor3, int motor4) {
     }
 }
 
+void Motor::go_all(int speeds[], int directions[]) {
+    for (int i = 0; i < sizeof(speeds) / sizeof(*speeds); i++)
+        Motor::go(i + 1, directions[i], speeds[i] = (speeds[i] < 0)?0:speeds[i]);
```

Comment:
Also, the call should only be `go`, not `Motor::go`, since `go` method is not static.